pyproject.toml

```diff
@@ -11,13 +11,13 @@ keywords = ["daml", "blockchain", "dlt", "distributed ledger", "digital asset"]
 python = ">=3.7, <4.0"
 aiohttp = "*"
 dacite = "*"
-dazl = "7.3.7"
+dazl = "7.4.0"
```

Comment:
I missed this before, but this should probably be `^7.4.0` (I don't think you want the pin so tight to this _specific_ version).

Reply:
I deliberately started tying to a specific version of DAZL when we got bit by some sort of regression in DAZL a few months ago. Happy to reconsider, but in the interests of engineering conservatism (appropriate for systems software), I still like binding specific integration frameworks to specific DAZL versions.